pkg/logs/config/config.go

```diff
@@ -19,6 +19,15 @@ import (
 // ContainerCollectAll is the name of the docker integration that collect logs from all containers
 const ContainerCollectAll = "container_collect_all"
 
+const logsAgentPrefix = "agent-intake.logs."
```

Comment:
nits: I'd rather call this `endpointPrefix` or something specific to the network layer as we are in the `config` package, it's not clear that it's a prefix for a domain name